src/content/docs/data-apis/ingest-apis/event-api/incident-event-rest-api.mdx

```diff
@@ -0,0 +1,310 @@
+---
+title: Incident event REST API
+tags:
+  - Alerts and Applied Intelligence
+  - Applied intelligence
+  - Incident intelligence
+metaDescription: Read about how to report activities from your incident management system to New Relic for advanced correlation.
+---
+
+You can use the incident event REST API to report custom events to New Relic.
+
+The incident API is built on our [event API](/docs/data-apis/ingest-apis/introduction-event-api).
+
+The incident event API is an asynchronous endpoint. This means you can send a large volume of POSTS, reliably, with low-response latency.
+
+## Using the API: an overview [#overview]
+
+Here's the general process for sending custom incident events to New Relic via the incident event API:
+
+1. Generate a [license key](/docs/apis/intro-apis/new-relic-api-keys/#ingest-license-key) for the account you want to report data to.
+2. Before you get started creating custom attributes, review the [event API's limits and restricted characters](/docs/data-apis/ingest-apis/introduction-event-api/#limits).
+3. Generate the JSON for your incident event that follows our [JSON format guidelines](/docs/data-apis/ingest-apis/introduction-event-api#instrument).
+4. Submit a compressed JSON payload (for example, gzip or deflate) to the HTTPS endpoint using curl in a POST request.
+
+Once you've started sending incident event data to New Relic, we recommend that you set up [NRQL alert conditions](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions) to get notifications about parsing errors.
+
+## Example JSON [#example-json]
+
+Your JSON payload may look something like this example.
+
+```json
+[
+  {
+    "eventType": "NrAiIncidentExternal",
+    "title": "Test",
+    "description": "The latency is above threshold of 500000 MS",
+    "state": "trigger",
+    "source": "luna",
+    "entityName": "testEntity",
+    "entity.guid": "testEntity123",
+    "aggregationTag.serviceId": 5,
+    "aggregationTag.environment": "testing",
+    "aggregationTag.errorId": 10543,
+    "tag.stackTrace": "some stack trace...",
+    "version": 1
+  }
+]
+```
+
+The `eventType` field must use `NrAiIncidentExternal`.
+
+## Send your incident event from the command line [#cli]
+
+Use curl to send your JSON payload.
+
+```bash
+gzip -c example_incidents.json | curl --data-binary @- -X POST -H "Content-Type: application/json" -H "X-Insert-Key: <var>YOUR_INSERT_KEY</var>" -H "Content-Encoding: gzip"
```

Comment:
```suggestion
gzip -c example_incidents.json | curl --data-binary @- -X POST -H "Content-Type: application/json" -H "Api-Key: <var>YOUR_LICENSE_KEY</var>" -H "Content-Encoding: gzip"
```

Reply:
This is related to change (which it'd be good to let the team know about) that the 'insert key' is now interchangeable with the license key and we should be using 'license key' as the preferred key for all ingest stuff. And use the 'Api-Key' header instead also.